training/coqui_stt_training/util/helpers.py

```diff
@@ -203,14 +219,20 @@ def pick_value_from_range(value_range, clock=None):
 
 def tf_pick_value_from_range(value_range, clock=None, double_precision=False):
     import tensorflow as tf  # pylint: disable=import-outside-toplevel
-    clock = (tf.random.stateless_uniform([], seed=(-1, 1), dtype=tf.float64) if clock is None
-             else tf.maximum(tf.constant(0.0, dtype=tf.float64), tf.minimum(tf.constant(1.0, dtype=tf.float64), clock)))
+    if clock is None:
+        clock = tf.random.stateless_uniform([], seed=(-1, 1), dtype=tf.float64)
+    else:
+        clock = tf.maximum(tf.constant(0.0, dtype=tf.float64),
+                           tf.minimum(tf.constant(1.0, dtype=tf.float64), clock))
     value = value_range.start + clock * (value_range.end - value_range.start)
-    value = tf.random.stateless_uniform([],
-                                        minval=value - value_range.r,
-                                        maxval=value + value_range.r,
-                                        seed=(clock * tf.int32.min, clock * tf.int32.max),
-                                        dtype=tf.float64)
+    if value_range.r:
```

Comment:
Why is this conditional being introduced?

Reply:
I added the `if` to make the logic more clear. Formerly, in the case `r==0`, the `tf.random.stateless_uniform` operation just returns `value`, unchanged. I think this `if` makes the code more readable.

Reply:
Yes, I see it. I was worried about changing the TF graph here and altering the behavior when the `clock` variable changes. But even when it does change I guess this call would still return `value` unchanged, so LGTM